app/interactors/importer.rb

```diff
@@ -118,6 +120,15 @@ def trips
 
   private
 
+  def gis_point!(obj, lat, lon, to:)
+    string_to_convert = "POINT(#{obj.delete(lat)} #{obj.delete(lon)})"
+    result = Stop.connection.execute <<-SQL
+      SELECT st_wkttosql('#{string_to_convert}');
```

Comment:
How do you feel about using [ST_GeomFromText](http://postgis.net/docs/manual-1.5/ST_GeomFromText.html) instead of `st_wkttosql`? It's up to you, but I feel like the former is clearer.